airbyte-integrations/connectors/source-facebook-pages/unit_tests/test_facebook_authenticator.py

```diff
@@ -19,13 +19,13 @@ def test_facebook_url_params(req_mock):
         "access_token": "initial_token",
         "page_id": "pageID"
     }
-    options = config
+    parameters = config
```

Comment:
The changes from `options` to `parameters` was part of https://github.com/airbytehq/airbyte/issues/19408 but some sources were not updated. I updated the sources that were impacted by my changes